internal/awsdata/data.go

```diff
@@ -116,6 +118,11 @@ func (d *AWSData) Load(regions, services []string) {
 
 	// Regional Services
 	for _, region := range regions {
+		if stringInSlice(ServiceDynamoDB, services) {
+			d.wg.Add(1)
```

Comment:
I've just merged some logging improvements to make reviewing PRs a easier, would you mind adding the new log line here?

https://github.com/manywho/awsinventory/blob/339a13886b7b50a894306138493eb3b54ca9466b/internal/awsdata/data.go#L121